.gitattributes

```diff
@@ -0,0 +1 @@
+go.sum -diff
```

Comment:
This won't take effect until merged but it'll exclude `go.sum` from the loc on PRs.

Reply:
Do you mean `go.mod`? `go.sum` is generated and just a bunch of checksums based on the `go.mod` dependency tree.

Reply:
If we don't want this I can remove it. I've done this on past projects to remove misleading line counts.

Reply:
I see. `k8s.io/client-go` and `sigs.k8s.io/controller-runtime` were added while a few other dependencies were removed. Quite a large number of indirect dependencies came from this, but I think these shouldn't be problems because Tink has roughly the same dependencies (differing on version).

Reply:
Yes. I was alarmed myself at the transitive dependencies introduced because it literally doubled but then realized it was mostly due to Kube pkgs.